django/contrib/postgres/fields/serial.py

```diff
@@ -0,0 +1,87 @@
+from django.core import checks
+from django.db import models
+from django.db.models.expressions import DatabaseDefault
+from django.utils.translation import gettext_lazy as _
+
+__all__ = ("BigSerialField", "SmallSerialField", "SerialField")
+
+
+class SerialFieldMixin:
+    db_returning = True
+
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault("default", DatabaseDefault())
+        super().__init__(
+            *args,
+            **{
+                **kwargs,
+                "blank": True,
+            },
+        )
```

Comment:
Assuming we want to force `blank=True` we can write this as:

```suggestion
        kwargs["blank"] = True
        super().__init__(*args, **kwargs)
```

Reply:
Yes, you're correct, this is how it's done in the `AutoFieldMixin` as well, I was just using an old branch that had this. I'll fix!